apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/blocks/TextResponse/TextResponse.stories.tsx

```diff
@@ -26,24 +28,16 @@ const TextResponseStory: Meta<typeof TextResponse> = {
   }
 }
 
-type Story = StoryObj<
+const Template: StoryObj<
   ComponentProps<typeof TextResponse> & { block: TreeBlock<TextResponseBlock> }
```

Comment:
Can we tidy this up while we're here?
```suggestion
  ComponentProps<typeof TextResponse>
```